src/components/FeatureStore/featureStore.util.js

```diff
@@ -345,14 +352,18 @@ export const handleFetchData = async (
 
     if (result) {
       data.content = parseFeatureSets(result)
-      data.yamlContent = result
+      data.originalContent = result
     }
   } else if (pageTab === FEATURES_TAB) {
-    result = await fetchFeatures(project, filters)
+    const allResult = await Promise.all([
```

Comment:
AS you can read [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all#description):

> `Promise.all()` will reject immediately upon any of the input promises rejecting. In comparison, the promise returned by `Promise.allSettled()` will wait for all input promises to complete, regardless of whether or not one rejects.

This means that if `fetchFeatures(project, filters)` fails, for example, then our `Promise.all` here will be rejected immediately and will not wait for `fetchEntities(project, filters)` to settle.

Now, I'm not sure what we really want to do here, but if we want to render a list with features only, in case we failed to fetch entities, and vice-versa, then we need to change `Promise.all` to `Promise.allSettled` (and later fetch the `value` of each object in the array returned to `allResult` variable). But maybe it makes more sense to just fail if _any_ of these lists failed to be fetched 🤔